migration/migration.go

```diff
@@ -410,6 +410,12 @@ func GetMigrations() Migrations {
 	// Version 90
 	m = append(m, steps{ExecuteSQLFile("090-queries-version.sql")})
 
+	// Version 91
+	m = append(m, steps{ExecuteSQLFile("091-comments-child-comments.sql")})
+
+	// Version 92
+	m = append(m, steps{ExecuteSQLFile("092-comment-revisions-child-comments.sql")})
```

Comment:
Why don't we combine 91 & 92 migrations into one? I don't see any particular benefit for two separate migrations.

Reply:
I tried that but it didn't work. It looks like the database version is checked and the migration is not re-applied. I was fearing that this also doesn't happen on deployment. If it doesn't this might be a source of hard to test issues.